### DISCUSSION TITLE
Use nette with tailwind ?

### DISCUSSION THREAD

**Discussion by Flashtheorie** [nette/forms#285] [Q&A]
Hi everyone, I'm wondering how to use nette but with the framework Tailwind ?
How to add classes etc to the inputs ?

**Comment by milo** [comment]
Hi. There are more ways. By custom form renderer (like in [this](https://github.com/nette/forms/blob/master/examples/bootstrap4-rendering.php) example for Bootstrap), or manually with manual rendering.

https://doc.nette.org/en/forms/rendering

**Reply by Flashtheorie** [reply]
Why is it so complicated just to add classes 🤔

**Reply by milo** [reply]
It is a universal way applycable to any form later. If you prefer simpler way, just add classes:
```php
$form->addText('name', ...)->setHtmlAttribute('class', '...');
```

**Reply by Flashtheorie** [reply]
This is actually just logical, sorry 😑
